examples/taskworkflow.md

```diff
@@ -0,0 +1,84 @@
+### About
+The `taskworkfow.go` file contains code to pull and unpack a container image
+using the `naive` snaphotter, create a container and execute a task using the
+`firecracker-containerd` runtime.
+
+### Building
+After checking out the `firecracker-containerd` repo, you can build this
+example using the `make examples` command.
+
+### Running
+For a basic workflow, without any networking setup, just run the executable
+built with the command in the previous section as super user:
+```bash
+$ sudo /path/to/firecracker-containerd/examples/taskworkflow
+```
+
+For a workflow with networking setup for the container, create a tap device
+for the VM by following [these instructions](https://github.com/firecracker-microvm/firecracker/blob/master/docs/network-setup.md).
+
+This will create a tap device named `tap0`. Now, edit the kernel args in the
+firecracker-containerd config to set a static ip address from the
+`172.16.0.1/24` subnet. The following example sets it to `172.16.0.2`.
+** NOTE: This example will not work if you're running more than 1 container
+on a host at the same time **
+```bash
+$ cat /etc/containerd/firecracker-runtime.json
+{
+    "firecracker_binary_path": "/home/ec2-user/firecracker-wrapper.sh",
```

Comment:
Is `firecracker-wraper.sh` something that should have been included in this PR?

Reply:
Nope, should be fixed now!